toss/fitness/fitness_function_utils.py

```diff
@@ -129,7 +149,14 @@ def compute_space_coverage(positions: np.ndarray, velocities: np.ndarray, timest
         # Compute the ratio of True values to the total number of values in the boolean tensor
         ratio = bool_tensor.sum() / bool_tensor.size
 
-        return ratio
+        # Define a zero-valued tensor with the same shape as the spherical meshgrid
+        r_idx = np.where(bool_tensor == True)[0]
+        sum_of_weights = np.sum(1/r[r_idx])
+
+        # Return fitness
+        fitness = ratio + sum_of_weights
+
+        return fitness
```

Comment:
it's a bit weird that you weight by adding numbers. also because this way coverage can be > 1 which makes it very difficult to interpret. but should be ok for the paper. but maybe create an issue for later